bazel/repositories.bzl

```diff
@@ -36,6 +36,14 @@ def proxy_wasm_cpp_host_repositories():
         urls = ["https://github.com/google/googletest/archive/release-1.10.0.tar.gz"],
     )
 
+    http_archive(
+        name = "wamr",
+        build_file = "@proxy_wasm_cpp_host//bazel/external:wamr.BUILD",
+        sha256 = "1d219d4121badc23b0743af2e7dc7e04348ace309748541deba897709f2d90b6",
+        strip_prefix = "wasm-micro-runtime-67fa1602031f5b86d195a3b1d02d331fcb70f299",
+        url = "https://github.com/bytecodealliance/wasm-micro-runtime/archive/67fa1602031f5b86d195a3b1d02d331fcb70f299.tar.gz",
```

Comment:
This PR (https://github.com/bytecodealliance/wasm-micro-runtime/pull/618) isn't merged yet.

Reply:
:laughing: We are working on JIT support now and gonna enable AOT in the next step.

Reply:
Oh, nice!

Reply:
Does WAMR need to merge this PR (bytecodealliance/wasm-micro-runtime#618)  first ?

Reply:
Ideally, yes. Otherwise, we're pulling some "random" commit that's not part of WAMR.

Reply:
It looks that it was merged (by someone else?) in bytecodealliance/wasm-micro-runtime#622.

Could you update this PR?

Reply:
Also, please merge `master` branch here to pickup other updates, otherwise your PR in Envoy won't work.

Reply:
Update to master now.